### PR TITLE
Solve connect to box

### PR DESCRIPTION
**Pull request by anbraten**
# Houston, we had a problem.

As our complete app is currently hosted on github.io it uses https. We then tried to use websocket connections to our local backends with none ssl connections, but most browsers block them with a "mixed content" error.

## Initial ideas:
- host backend with ssl => difficult as we need to get let's encrypt certificates which is impossible without port-forwarding or dns access

## Final solution:
Split the app. We now have the entrypoint / setup hosted on github.io with an easy to remember address. On that page you can enter you boxId (currently local IP of the box) and click on connect which simply redirects you to a ssl-less http website on the local ip of your box hosting the same app, but which behaves as the "normal" app.
To host the app on the box as well, we now have an additional docker image `leek-box`.

### CODE REVIEW

File: packages/app/src/views/Setup.vue [hunk 1]
```diff
@@ -45,25 +37,22 @@ export default defineComponent({
   },
 
   setup() {
-    const router = useRouter();
-    const backendUrl = ref('');
+    const boxId = ref('');
```
Comment (schnickalodeon) [inline packages/app/src/views/Setup.vue:40]: No type (ref<string>(''); 

Doesn't that fire a linting error?

**Review by schnickalodeon — CHANGES_REQUESTED** [review]
lgtm 

Just a short question...

File: packages/app/src/views/Setup.vue (hunk 1 above)
Comment (anbraten) [reply packages/app/src/views/Setup.vue:40]: It implicitly takes the type from the empty string.

**Review by schnickalodeon — APPROVED** [review]
nice solution :D

**Review by julianhahn — APPROVED** [review]
looks good

**Pull request merged**